### PR TITLE
crypto++: new recipe

### PR DESCRIPTION
**Pull request by return**
This is needed to build [Librevault](https://github.com/Librevault/librevault/blob/master/docs/build.md). Should be able to compile with any C++11 compiler.

### CODE REVIEW

File: dev-libs/cryptopp/cryptopp-5.6.5.recipe [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* Block cipher modes of operation: ECB, CBC, CBC (CTS), CFB, OFB
+* Message authentication codes: VMAC, HMAC, GMAC, CMAC, CBC-MAC, DMAC
+* Hash functions: BLAKE2 (BLAKE2b, BLAKE2s), SHA-1, SHA-2, SHA-3
+* Public-key cryptography: RSA, DSA, ElGamal, Nyberg-Rueppel (NR)
+* Padding schemes for public-key systems: PKCS#1 v2.0, OAEP, PSS,
+* Key agreement schemes: Diffie-Hellman (DH), Unified Diffie-Hellman (DH2),
+* Elliptic curve cryptography: ECDSA, ECNR, ECIES, ECDH, ECMQV
+* Insecure or obsolescent algorithms: MD2, MD4, MD5
+* And many more other algorithms.
+"
```
Comment (waddlesplash) [inline dev-libs/cryptopp/cryptopp-5.6.5.recipe]: Move to previous line

File: dev-libs/cryptopp/cryptopp-5.6.5.recipe [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	cp -r *.h $includeDir/cryptopp
+
+	packageEntries devel \
+			$developDir
+}
+
+TEST()
+{
+	make test
+}
```
Comment (waddlesplash) [inline dev-libs/cryptopp/cryptopp-5.6.5.recipe]: Should have newline at end of file.

**Comment by korli** [comment]
Should be crypto++ according to gentoo portage.

**Comment by return** [comment]
@korli @waddlesplash Applied suggested changes to recipe.

**Pull request merged**